internal/validate/kube/utils.go

```diff
@@ -0,0 +1,49 @@
+package kube
+
+import (
+	"fmt"
+	"path/filepath"
+	"reflect"
+	"strings"
+
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+func Contains(sl *[]string, t string) bool {
+	for _, s := range *sl {
+		if s == t {
+			return true
+		}
+	}
+	return false
+}
+
+// checks if current context is set to EKS cluster
```

Comment:
Comment should start with function name in this case, but this likely doesn't need to be exported anyhow